hardware/display/README.md

```diff
@@ -6,15 +6,35 @@ The Raspberry Pi Touch Display is an LCD display which connects to the Raspberry
 
 ## Board support
 
-The DSI display is designed to work with Raspberry Pi models 2, 3 and 4 that have mounting holes in a HAT footprint. Model 1 A/B boards are supported, but require additional mounting hardware to fit the HAT-dimensioned stand-offs on the display PCB.
+The DSI display is designed to work on all models of Raspberry Pi, however early models that do not have mounting holes (The Raspberry Pi 1 model A and B) will require additional mounting hardware to fit the HAT-dimensioned stand-offs on the display PCB.
```

Comment:
"on all models" -> "with all models" ?
Lowercase 'T' on "The Raspberry Pi"

Sorry @JamesH65 :laughing: 

Wasn't there also something to do with the I2C / DSI connections being different on pre-40-pin-GPIO models of Raspberry Pi, or am I confusing that with something else?

Reply:
You need to connect to the I2C on the GPIO on those earlier models. Its covered on the legacy page.

Reply:
Ooops, I missed the link to that page :eyes: